helix-view/src/theme.rs

```diff
@@ -42,17 +45,41 @@ impl Loader {
         if name == "base16_default" {
             return Ok(self.base16_default());
         }
-        let filename = format!("{}.toml", name);
 
-        let user_path = self.user_dir.join(&filename);
-        let path = if user_path.exists() {
-            user_path
+        let theme_toml = self.load_theme(name, name, false)?;
+
+        Ok(Theme::from(theme_toml))
+    }
+
+    // load the theme and its parent recursively and merge them
+    // `base_theme_name` is the theme from the config.toml,
+    // used to prevent some circular loading scenarios
+    fn load_theme(
+        &self,
+        name: &str,
+        base_them_name: &str,
+        only_default_dir: bool,
+    ) -> Result<Value, anyhow::Error> {
+        let path = self.path(name, only_default_dir);
+        let theme_toml = self.load_toml(path)?;
+
+        let inherits_from = theme_toml.get("inherits_from");
```

Comment:
Let's just call this `inherits` which we use already in the tree-sitter queries

Reply:
will do